cfgov/jinja2/v1/youth_employment_success/route-questions/driving-cost-estimate.html

```diff
@@ -1,5 +1,5 @@
 <div class="content-l block block__sub js-driving-estimate">
-  <div class="content-l_col content-l_col-1">
+  <div class="content-l_col content-l_col-1 a-label">
```

Comment:
Hmm, this structure seems odd:

```
div.a-label
  p.a-label
```

Any other way to do this?

Reply:
Oh sure, I should be able to just take the a-label off of the `p` tag!